### DISCUSSION TITLE
Help on DepthAI jar which is not supported for windows

### DISCUSSION THREAD

**Discussion by mksbabu** [bytedeco/javacpp-presets#1428] [Q&A] [answered]
Hi Team,

I am using below jar for DepthAI in java code and getting below error, could you please anyone help me on it

**Jar:** 
<dependency>
            <groupId>org.bytedeco</groupId>
            <artifactId>depthai-platform</artifactId>
            <version>2.21.2-1.5.9</version>
        </dependency>
**Error:**
Exception in thread "main" java.lang.UnsatisfiedLinkError: Platform "windows-x86_64" not supported by class org.bytedeco.depthai.Pipeline
	at org.bytedeco.javacpp.Loader.load(Loader.java:1261)
	at org.bytedeco.javacpp.Loader.load(Loader.java:1234)
	at org.bytedeco.javacpp.Loader.load(Loader.java:1210)
	at org.bytedeco.depthai.Pipeline.<clinit>(Pipeline.java:25)
	at code.DepthAICamera.main(DepthAICamera.java:8)

**Comment by saudet** [comment]
I don't believe DepthAI supports Windows. @themarpe?

**Reply by mksbabu** [reply]
DepthAI code is supporting with c++ in windows, I hope DepthAI java will also support in windows, Could you please anyone help us on it.

**Reply by saudet** [reply]
I see, contributions are welcome!

**Reply by mksbabu** [reply]
@saudet I have seen similar type of issue(https://github.com/bytedeco/javacpp-presets/issues/983), it was fixed by you after adding windows in yml file, could you please help me on it.

**Reply by mksbabu** [reply]
Hi @saudet @themarpe

could you please make a change for DepthAI(https://github.com/bytedeco/javacpp-presets/issues/983) to support for windows, It will be great help.

**Reply by saudet** [reply]
Sure, like I said contributions are welcome! Please open a pull request with the changes you want.

**Reply by mksbabu** [reply]
Hi @saudet 

created pull request, could you please look into it, let me know please if any changes are required to support depthai in windows

pull request: https://github.com/bytedeco/javacpp-presets/pull/1430

Thanks.

**Reply by themarpe** [reply]
DepthAI does support Windows, so from that perspective all should be fine

**Reply by mksbabu** [reply]
Hi @saudet @themarpe

I am facing few build issues with this project when I imported in my local, not able to proceed further. If you don't mind, could you please make depthai to work in windows. I am thankful for help.

Thanks.

**Reply by themarpe** [reply]
@mksbabu depthai (C++/Python) works on Windows as expected. For getting the Java build over, I'm afraid I can only glance over some compilation logs and provide guidance, but we are not actively keeping Java bindings up to date

**Reply by mksbabu** [reply]
@themarpe  Do you mean, It will be difficult to use depthai by windows?

**Reply by saudet** [reply, marked as answer]
Thanks for @richard-xx, we now have DepthAI builds for Windows! Enjoy
